README.md

```diff
@@ -1,45 +1,31 @@
 # IHMC Open Robotics Software
 
-**Compile:** ![Compile](https://bamboo.ihmc.us/plugins/servlet/wittified/build-status/LIBS-IHMCOPENROBOTICSSOFTWARE)
-**Test (3000+ tests):** ![Test](https://bamboo.ihmc.us/plugins/servlet/wittified/build-status/LIBS-IHMCOPENROBOTICSSOFTWAREFAST)
+## Robots
 
-### Tested Platforms
-
-#### Robots
-
-- DRC-Atlas
```

Comment:
Does this mean we should remove the Atlas related code?

Reply:
No lol